ci.config.json

```
@@ -11,6 +11,12 @@
       "estuary": "0.1.1",
       "cross": "0.2.5",
       "toml-cli2": "0.3.2"
+    },
+    "git": {
+      "estuary": {
+        "url": "https://github.com/fuzzypixelz/estuary.git",
```

```suggestion
        "url": "https://github.com/ZettaScaleLabs/estuary.git",
```

I added a fork under ZettaScaleLabs so we don't have to rely on my private account 🙃